CMakeLists.txt

```diff
@@ -219,37 +219,37 @@ endif()
 # Get version
 #
 
-set (PACKAGE_VERSION "")
```

Comment:
Whether `PACKAGE_VERSION` is dropped intentionally? It is used in `src/trivia/config.h.cmake`.

Reply:
Hm. It seems, it is something about packaging using CPack. We don't use it AFAIK.

Reply:
It's set right below anyway so yes, I dropped it intentionally:

https://github.com/tarantool/tarantool/blob/3f3bb245cc792358ad037efddbf8d643001115e5/CMakeLists.txt#L271